### DISCUSSION TITLE
FPS I have been working on with the library

### DISCUSSION THREAD

**Discussion by N8python** [enable3d/enable3d#108] [Show and tell]
https://user-images.githubusercontent.com/43304488/107542581-562d2e00-6b96-11eb-932d-bf7e15aafd38.mov

This is what all the questions were about lol. Thanks for the awesome library and tools!

**Comment by vitalspace** [comment]
nice work

**Reply by N8python** [reply]
Thanks!

**Comment by yandeu** [comment]
Looks great 😁👍🏻

**Reply by N8python** [reply]
Thanks!
